probeinfo_api.yaml

```diff
@@ -0,0 +1,562 @@
+      properties:
+        v1_name:
+          $ref: "#/components/schemas/V1Name"
+        description:
+          $ref: "#/components/schemas/Description"
+        deprecated:
+          $ref: "#/components/schemas/DeprecatedBool"
+        notification_emails:
+          $ref: "#/components/schemas/NotificationEmails"
+        url:
+          $ref: "#/components/schemas/RepoUrl"
+        branch:
+          $ref: "#/components/schemas/RepoBranch"
+        metrics_files:
+          $ref: "#/components/schemas/MetricsFiles"
+        ping_files:
+          $ref: "#/components/schemas/PingFiles"
+        library_names:
+          $ref: "#/components/schemas/LibraryNames"
+    Application:
+      type: object
+      additionalProperties: false
+      required:
+        - app_id
+        - document_namespace
+        - bq_dataset_family
+        - app_name
+        - v1_name
+        - url
+      properties:
+        app_id:
+          $ref: "#/components/schemas/AppId"
+        document_namespace:
+          $ref: "#/components/schemas/DocumentNamespace"
+        bq_dataset_family:
+          $ref: "#/components/schemas/BqDatasetFamily"
+        app_name:
+          $ref: "#/components/schemas/AppName"
+        canonical_app_name:
+          $ref: "#/components/schemas/CanonicalAppName"
+        v1_name:
+          $ref: "#/components/schemas/V1Name"
+        app_channel:
+          $ref: "#/components/schemas/AppId"
+        description:
+          type: string
+        notification_emails:
+          $ref: "#/components/schemas/NotificationEmails"
+        url:
+          $ref: "#/components/schemas/RepoUrl"
+        branch:
+          $ref: "#/components/schemas/RepoBranch"
+        metrics_files:
+          $ref: "#/components/schemas/MetricsFiles"
+        ping_files:
+          $ref: "#/components/schemas/PingFiles"
+        dependencies:
+          $ref: "#/components/schemas/Dependencies"
+        deprecated:
+          $ref: "#/components/schemas/DeprecatedBool"
+        prototype:
+          $ref: "#/components/schemas/PrototypeBool"
+        retention_days:
+          $ref: "#/components/schemas/RetentionDays"
+    V1Name:
+      type: string
+      pattern: "^[a-z][a-z0-9-]{0,29}$"
+      description: Identifier used in v1 URI paths that reference a specific repository
+      example: firefox-android-beta
+    Description:
+      type: string
+      description: A brief free-text description of the repository
+    AppName:
+      type: string
+      pattern: "^[a-z][a-z0-9_]{0,39}$"
+      description: |
+        A lowercase_with_underscores name that is short and unambiguous. It needs to be something that can be used both in a generated table name in BigQuery or as a value in an `app_name` string field in a derived table. BigQuery users might be using this literal name in a `WHERE` clause, so it should feel reasonable for that use case. Examples: `fenix`, `firefox_ios`
+      example: firefox_ios
+    CanonicalAppName:
```

Comment:
We should perhaps add more newlines in this file and repositories.yaml per discussion in https://github.com/mozilla/probe-scraper/pull/260#discussion_r553364614